docs/adapters/index.md

```diff
@@ -12,6 +12,7 @@ These adapters are included in the Juttle runtime and can be used in Juttle prog
 
 * [file](../adapters/file.md)
 * [http](../adapters/http.md)
+* [io](../adapters/io.md)
```

Comment:
I'd find `io` to be too generic of a name for stdin / stdout. Every adapter is technically some kind of input/output.

I know this is contrary to what @go-oleg suggested, but I find `read stdin` and `write stdout` to be more intuitive than `read io`.